snugget_load.py

```diff
@@ -28,25 +29,57 @@ def main():
     with conn.cursor() as cur:      
       with open(snuggetFile) as csvFile:
         newSnuggets = csv.DictReader(csvFile)
+        rowCount = 1
```

Comment:
Row 1 in the CSV file is the headings, so row 2 is the first data row. Should I add a comment clarifying that?